validator/client/src/test/java/tech/pegasys/teku/validator/client/ValidatorRegistratorTest.java

```diff
@@ -381,6 +398,11 @@ private void verifyRegistrations(
             .getBuilderRegistrationTimestampOverride()
             .orElse(stubTimeProvider.getTimeInSeconds());
 
+    final UInt64 expectedPubkey =
+        validatorConfig
+            .getBuilderRegistrationPubkeyOverride()
+            .orElse();
+
```

Comment:
line 398 in this file handles the testing of pubkey

ValidatorRegistration::getPublicKey

appears to do it for multiple pubkeys, not sure how this test should be altered for the pubkey-override flag